packages/components/data-table/src/data-table.tsx

```diff
@@ -239,8 +239,15 @@ export type TDataTableProps<Row extends TRow = TRow> = {
 };
 
 const DataTable = <Row extends TRow = TRow>(props: TDataTableProps<Row>) => {
+  const columns =
```

Comment:
if we use this alternative, we would miss the initial state. and it seems to me like we simply just need to pass the columns object just the same way it was used in the data-table-manager. 
```suggestion
const [columns, setColumns] = useState<[]>();
  const handleCustomEvent = (event) => {
      const { columns } = event.detail;
      setColumns(columns);
    };
    useEffect(() => {
      window.addEventListener(‘DataTaBleManagerEvent’, handleCustomEvent);
      return () => {
        window.removeEventListener(‘DataTaBleManagerEvent’, handleCustomEvent);
      };
    }, []);